src/Workspaces/Core/Portable/Rename/RenameOptions.cs

```diff
@@ -2,44 +2,33 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
+using System;
+using System.Runtime.Serialization;
 using Microsoft.CodeAnalysis.Options;
 
 namespace Microsoft.CodeAnalysis.Rename
 {
+    [Obsolete("Use SymbolRenameOptions or DocumentRenameOptions instead")]
```

Comment:
slightly oogy on this.  say someone doesn't move over.  what's the net negative impact for them?  or is the intent that this is just totally broken and we need to remove it in the future and have to start warning people now?

Reply:
The options here can still be used with the obsolete Renamer.RenameSymbolAsync overloads. We'll keep the API for a while, but at some point we might want to remove it to clean up.